app/views/admin/characters/_relocate.haml

```diff
@@ -15,10 +15,10 @@
         %th.sub= f.label :user_id, 'New User Id'
         %td{class: cycle('even', 'odd')}= f.text_field :user_id, value: params[:user_id], placeholder: "New User", class: 'text'
       %tr
-        %td.sub With Templates
+        %th.sub With Templates
```

Comment:
(this also fixes some `%td.sub` to `%th.sub` in admin pages, which improves some formatting there)